src/Stratis.Bitcoin.Features.PoA.Tests/PoATestsBase.cs

```diff
@@ -109,18 +109,23 @@ public class TestPoANetwork : PoANetwork
     {
         public TestPoANetwork(List<PubKey> pubKeysOverride = null)
         {
-            var federationPublicKeys = new List<PubKey>()
+            var genesisFederation = new List<IFederationMember>()
```

Comment:
Why are we first constructing the list before checking the `pubKeysOverride`? Seems a bit in efficient?

Reply:
Fixed